src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/12.diff.sql

```diff
@@ -0,0 +1,271 @@
+﻿
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+
+This diff is broken up into several sections:
+ - The first transaction contains changes to tables and stored procedures.
+ - The second transaction contains updates to indexes.
+ - IMPORTANT: Avoid rebuiling indexes inside the transaction, it locks the table during the transaction.
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+
+DROP INDEX IF EXISTS IXC_Workitem ON dbo.Workitem
+GO
+CREATE UNIQUE CLUSTERED INDEX IXC_Workitem ON dbo.Workitem
+(
+    WorkitemKey
+)
+GO
+
+DROP INDEX IF EXISTS IX_Workitem_WorkitemKey_Watermark ON dbo.Workitem
```

Comment:
Why do this way, since you are adding new. CHECK FOR IF NOT EXISTS and then create

Reply:
Done